src/bench/checkqueue.cpp

```diff
@@ -26,6 +26,9 @@ static const unsigned int QUEUE_BATCH_SIZE = 128;
 // and there is a little bit of work done between calls to Add.
 static void CCheckQueueSpeedPrevectorJob(benchmark::Bench& bench)
 {
+    // We shouldn't ever be running with the checkqueue on a single core machine.
+    if (GetNumCores() < MIN_CORES) return;
```

Comment:
Maybe ditch `MIN_CORES` and change this to `GetNumCores() == 1`?

Reply:
https://en.cppreference.com/w/cpp/thread/thread/hardware_concurrency:
> If the value is not well defined or not computable, returns ​`0`​.

Reply:
Maybe `<=1` ?

Reply:
Thanks! [Updated](https://github.com/bitcoin/bitcoin/pull/19710#issuecomment-680785136).